test/easyconfigs/easyconfigs.py

```diff
@@ -1778,6 +1778,13 @@ def template_easyconfig_test(self, spec):
                 fail_msg += f"'{regex.pattern}'"
                 failing_checks.append(fail_msg)
 
+    # meson buildtype should be specified with easyblock parameter "buildtype" not with custom configopts.
+    if ec['easyblock'] == 'MesonNinja':
+        configopts = ec.get('configopts', '', resolve=True)
+        if '--buildtype ' in configopts:
```

Comment:
@Micket Should also consider `--buildtype=`?
```suggestion
        if '--buildtype ' in configopts or '--buildtype=' in configopts: